pkg/resources/server_test.go

```diff
@@ -413,5 +414,47 @@ var _ = Describe("Server", func() {
 				close(done)
 			}, 30.0)
 		})
+		Context("when CDI is enabled", func() {
```

Comment:
can you also add a test case for Allocate when CDI is enabled ?

Reply:
done